index.js

```diff
@@ -17,15 +17,18 @@ function heightAscThanNameComparator(a, b) {
 
 /**
  * Pack a list of images with width and height into a sprite layout.
+ * options object with the following keys:
  *
  * @param   {Object[]}              imgs        Array of `{ svg: Buffer, id: String }`
  * @param   {number}                pixelRatio  Ratio of a 72dpi screen pixel to the destination pixel density
  * @param   {boolean}               format      If true, generate {@link DataLayout}; if false, generate {@link ImgLayout}
  * @param   {Function}              callback    Accepts two arguments, `err` and `layout` Object
+ * @param   {boolean}               maxIconSize optional, overrides the max_size in mapnik
  * @return  {DataLayout|ImgLayout}  layout      Generated Layout Object with sprite contents
  */
-function generateLayout(imgs, pixelRatio, format, callback) {
-    return generateLayoutInternal(imgs, pixelRatio, format, false, callback);
+function generateLayout(options, callback) {
+    options.unique = false;
```

Comment:
I'm a bit paranoid, so I like to add:
```js 
options = options || {};
```
...to the beginning of the function - just in case somebody passes in a null/undefined `options` argument.  Then you can still do things like `options.unique` without risking a "ReferenceError: options is not defined".